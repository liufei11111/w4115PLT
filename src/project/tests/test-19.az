{* option access *}
Int main(Int argc, String argv)｛
String str;
str = "str";
Option o = { a=str, b="b"};
o->a = o->b;
｝