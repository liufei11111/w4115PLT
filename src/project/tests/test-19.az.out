stmts:

string str;
str = "str";
Option o = {a = str, b = "b"};
o->a = o->b;

fdecls:



